main.py

```
@@ -407,6 +407,96 @@ def main():
+                        return
+                    sanitized_args.extend([arg, out_file])
+                    i += 2
+                else:
+                    sanitized_args.append(arg)
+                    i += 1
+            # Locate ocrs_wrapper.py
+            wrapper_path = os.path.join(os.path.dirname(__file__), "ocrs_wrapper.py")
+            if not os.path.isfile(wrapper_path):
+                print(f"Error: ocrs_wrapper.py not found at {wrapper_path}")
+                return
+            cmd = [wrapper_path] + sanitized_args
```

Ruff (0.13.1)

Consider `[wrapper_path, *sanitized_args]` instead of concatenation

Replace with `[wrapper_path, *sanitized_args]`

(RUF005)